### DISCUSSION TITLE
Search Check-Ins

### DISCUSSION THREAD

**Discussion by PianoRailways** [Traewelling/traewelling#2018] [Ideas / Feature Requests]
To find older check-ins, it would be really helpful, if you can sort your check-ins by point of deparure, destination or type of vehicle (tram, bus, train etc.).

Until now, I have to remember when was my journey and click through the hundreds of pages. It would be already a big step, if you can search a journey by date (check-ins before/after a specific date ...)

**Comment by pascalwittler** [comment]
Mögliche Überschneidung mit #2779
